utils/prezly/getPrezlyApi.ts

```diff
@@ -8,7 +8,7 @@ const getPrezlyApi = (req: IncomingMessage, env: Env = getEnvVariables(req)!): P
         throw new Error('"getPrezlySdk" should only be used on back-end side.');
     }
 
-    return new PrezlyApi(env.PREZLY_ACCESS_TOKEN);
+    return new PrezlyApi(env.PREZLY_ACCESS_TOKEN, parseInt(env.PREZLY_NEWSROOM_ID));
```

Comment:
Aren't we using UUIDs? In that case parseInt would not work

Reply:
I changed the query to filter by newsroom to use the UUID, but I kept the ID here because the sdk `getNewsroom` works with IDs only.

Reply:
In this case please file a bug for the SDK team or @e1himself to have a look at. 

We can't be argumenting to use UUIDs in one place, and have IDs in another without a task/TODO to address this

Reply:
Filed a bug: DEV-2388
Will sort it out asap.

Reply:
The fix has been published within `prezly/javascript-sdk` v2.5.0